public/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0,user-scalable=no">
```

Comment:
Very user unfriendly. Was this a specific client request?
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#Viewport_scaling for reference

Reply:
Not really, it's a hack to disable the viewport automatically zooming in on focused inputs and selects. As I understand it, there is a 16px font-size threshold for that to NOT happen on iOS devices. My native select box uses a font size of 0 to completely hide its content and replace it with a background image... If you know of any other solutions to this, I'd be happy to remove that user-scalable property right there :)

Reply:
alright, I found another fix for this that did not have me hacking the viewport. Resolved!